omod-1.8/src/test/java/org/openmrs/module/webservices/rest/web/v1_0/controller/openmrs1_8/PatientIdentifierTypeController1_8Test.java

```diff
@@ -130,6 +134,23 @@ public void shouldRetireAPatientIdentifierType() throws Exception {
 		assertEquals(reason, service.getPatientIdentifierTypeByUuid(getUuid()).getRetireReason());
 	}
 	
+	@Test
+	public void shouldUnRetireAPatientIdentifierType() throws Exception {
+		PatientIdentifierType patientIdentifierType = service.getPatientIdentifierTypeByUuid(getUuid());
+		patientIdentifierType.setRetired(true);
+		patientIdentifierType.setRetireReason("random reason");
+		patientIdentifierType = service.savePatientIdentifierType(patientIdentifierType);
```

Comment:
Generally, instead of asserting using the returned object, i would prefer fetching it afresh using getPatientIdentifierTypeByUuid and then assert that it is retired

Reply:
I agree @dkayiwa . Let me refactor to reflect these changes.